appcues/src/main/java/com/appcues/statemachine/StateMachine.kt

```diff
@@ -174,7 +176,7 @@ internal class StateMachine(
 
             // Paused
             state is Paused && action is Resume ->
-                Transition(state.state)
+                Transition(state.state, emitStateChange = false)
```

Comment:
setting this flag `emitStateChange = false` on return from Resume will avoid analytics issues.  I don't see any reason why resets from resume need to be broadcast, as its really just used to reset the internal state.  I didn't go this far (yet) but the `Paused` state transition really seems it doesnt need to be emitted as well.